src/Core/Commander/Providers/Fetcher.js

```diff
@@ -2,6 +2,8 @@ import { TextureLoader } from 'three';
 
 const textureLoader = new TextureLoader();
 
+textureLoader.setCrossOrigin('');
```

Comment:
> So it suits you for this PR?

As far as I'm concerned, yes. Maybe use `anonymous` instead of the empty string to make it more explicit?